tests/templates/test_subroutines/test_qsvt.py

```diff
@@ -72,11 +72,11 @@ def test_init_error(self):
                 [qml.RZ(-2 * theta, wires=0) for theta in [1.23, -0.5, 4]],
                 [0],
                 [
-                    qml.RZ(-8, wires=0),
+                    qml.RZ(-2.46, wires=0),
```

Comment:
Why this test should be changed?

Reply:
The order of the operations was backwards.
It didn't fail before because the operations commute (?).

Reply:
Then the order in the previous tests should be changed as well? e.g., line 62 and lines 65-67.

Reply:
Lines 62 and 65-67 were in the correct order.

The order in `lst_projectors` and `operations` should be the same. These both refer to circuit order.